Signal/src/views/NotificationView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+class NotificationView: UIView {
```

Comment:
I hear this is one of those "hard" problems. I've settled on "ReminderView" since InfoMessage might get conflated with the "Info Messages" that occur in the MessagesView.